homework/week-2/newStyle.css

```diff
@@ -0,0 +1,100 @@
+.header {
+  background-color: #176ab0;
+  color: white;
+  text-align: center;
+  padding: 5px;
+}
+
+#main-content {
+  width: 350px;
+  float: left;
+  padding: 10px;
+}
+
+#footer {
+  background-color: #176ab0;
+  color: white;
+  clear: both;
+  text-align: center;
+  padding: 5px;
+}
+
+.badge {
```

Comment:
This doesn't work well when there are more than 100 contributions, don't forget to test with larger amounts of data!